docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Automatically Refresh Inheritance Timelocks with Payjoin"
+description: "Liana combines Payjoin and time locked inheritance to help you keep your Bitcoin private now and safe for those who’ll need it later."
+date: "2025-04-10"
+authors:
+  - Armin Sabouri
+tags:
+  - privacy
+  - integrations
+---
+
+[Liana](https://wizardsardine.com/liana/) is a Bitcoin desktop wallet that enables users to set up inheritance policies, ensuring their funds can be passed on to loved ones only after a predefined delay called a "timelock". To prevent unintentional activation of the inheritance policy, these time locked coins require periodic refreshing. Liana’s Payjoin integration enables users to refresh timelocks as they receive payments, boosting privacy and cutting costs.
+
+## Privacy Leak Explained
+
+Inheritance planning is a crucial yet often overlooked part of Bitcoin self-custody. While Bitcoin offers unmatched financial sovereignty, it introduces [unique challenges](https://wizardsardine.com/blog/the-state-of-bitcoin-inheritance-planning/) when it comes to passing on funds to future generations. Liana addresses this with inheritance policies that activate only after a timelock. For example, a user might control a primary key for daily use, while a recovery key held by a beneficiary is time locked for future access.
+
+However, there's a catch: most users don't frequently spend their Bitcoin. [A 2024 study](https://www.sciencedirect.com/science/article/pii/S0378437124008045) shows the age of unspent coins follows a power-law distribution. In other words, coins are either spent quickly or left untouched for long periods. [Data also suggests](https://charts.bitbo.io/hodl-waves/) that most coins remain unspent for months or years. To avoid unintentionally triggering a recovery path, users often resort to self-spending or reconciliation to reset timelocks.
```

Comment:
show => cite the figure?

Reply:
is self-spend different from reconciliation? Why 2 pieces of vocab?

Reply:
Fixed. And cited the more relevant figure. Thanks for pointing that out